datasets/il/mod_crypto/il_mod_crypto.yml

```
@@ -0,0 +1,28 @@
+title: Israel Sanctioned Crypto Wallets List
+entry_point: crawler.py
+prefix: il-nbctf
+coverage:
+  frequency: daily
+  start: 2021-10-22
+deploy:
+  schedule: "0 */6 * * *"
```

I think you can make it run weekly instead of 6-hourly since it only needs to pick up data model changes or if someone goes and updates the csv

```suggestion
deploy:
  schedule: "@weekly"
```